src/openforms/scss/vendor/_django-two-factor-auth.scss

```
@@ -0,0 +1,6 @@
+body.login.centered-image {
```

```suggestion
body.login .centered-image {
```
the space is important because the classname is applied to a _child of_ the body, not to the body itself.